### PR TITLE
Nitfixing24

### PR DESCRIPTION
**Pull request by SamYuan1990**
- FAQ in error msg
- fabric 2.4 in CI
- doc update

### CODE REVIEW

**Comment by SamYuan1990** [comment]
fix ci error

File: test/bootstraps/bootstrap-v2.4.sh [hunk 1]
```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
```
Comment (davidkhala) [inline test/bootstraps/bootstrap-v2.4.sh:1]: 这一段是从fabric-sample那复制过来的么？

File: test/integration-test.sh [hunk 2]
```diff
@@ -62,6 +62,26 @@ case $1 in
       ARGS=(-cci initLedger)
     fi
 
+    echo y |  ./network.sh deployCC -ccn basic -ccp ../asset-transfer-basic/chaincode-go/ -ccl go "${ARGS[@]}"
+    ;;
+ 2_4)
+    # Why comment following code? Please check this issue: https://github.com/Hyperledger-TWGC/tape/issues/159
+    # curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/release-2.3/scripts/bootstrap.sh | bash
```
Comment (davidkhala) [inline test/integration-test.sh:69]: 我第一眼看这一段有点懵，后来明白了，要不这样写? 合并成一行
# We intentionally do not use `curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/release-2.3/scripts/bootstrap.sh | bash` due to tape issue #159

**Comment by davidkhala** [comment]
这里拿不到 ${{ secrets.TWGC_DAVIDKHALA }}有点神奇，我在[这里]开个sancheck 检查下
github.com/Hyperledger-TWGC/Hyperledger-TWGC

**Review by davidkhala — DISMISSED** [review]

File: test/bootstraps/bootstrap-v2.4.sh (hunk 1 above)
Comment (SamYuan1990) [reply test/bootstraps/bootstrap-v2.4.sh:1]: 是

**Comment by guoger** [comment]
LGTM

**Pull request merged**